polyfill/lib/plaintime.mjs

```diff
@@ -66,12 +66,12 @@ function TemporalTimeToString(time, precision, options = undefined) {
 
 export class PlainTime {
   constructor(isoHour = 0, isoMinute = 0, isoSecond = 0, isoMillisecond = 0, isoMicrosecond = 0, isoNanosecond = 0) {
-    isoHour = ES.ToInteger(isoHour);
-    isoMinute = ES.ToInteger(isoMinute);
-    isoSecond = ES.ToInteger(isoSecond);
-    isoMillisecond = ES.ToInteger(isoMillisecond);
-    isoMicrosecond = ES.ToInteger(isoMicrosecond);
-    isoNanosecond = ES.ToInteger(isoNanosecond);
+    isoHour = ES.ToFiniteInteger(isoHour);
```

Comment:
where is this AO defined?

Reply:
See https://github.com/tc39/proposal-temporal/pull/1554#discussion_r657388876

Reply:
why add it to the polyfill before there's spec text for it?